src/auth/registration/RegistrationContainer.tsx

```diff
@@ -18,17 +18,19 @@ export function RegistrationContainer(
     <Background>
       <LandingPageAppBar />
       <Toolbar />
-      <PaddedCardContainer>
-        <img src={logo} height={50} />
-        {!isMobile && <div style={{ width: 600 }} />}
-        <Typography variant="h5" mt={3} mb={2}>
-          <Trans>Start your company profile</Trans>
-        </Typography>
+      <Box p={1}>
+        <PaddedCardContainer>
+          <img src={logo} height={50} />
+          {!isMobile && <div style={{ width: 400 }} />}
```

Comment:
nit: a better way to do this would be to use the break-point based xs prop to set the margins in the component below, something like:

`sx={{ width: { md: 400 } }}` (not checked exact syntax)

Because that is implemented 100% in css by a media query rather than in js.

[example](https://github.com/brave/ads-admin-dashboard/blob/3523ca0427413f44b57b190790d14fdf15159daa/src/routes/search/eligibility/DomainEligible.tsx#L57)